clodius/cli/aggregate.py

```diff
@@ -451,8 +451,10 @@ def line_to_np_array(line):
             importance = int(line[int(importance_column)-1])
 
         if stop < start:
-            print("WARNING: stop < start:", line, file=sys.stderr)
-            return
+            # print("WARNING: stop < start:", line, file=sys.stderr)
```

Comment:
If at least one of these is found, it should definitely print a warning (if not throw an error and force the user to fix the input themselves). In most cases, having stop < start won't be intentional.